test/commands.js

```diff
@@ -316,9 +316,9 @@ let runFinalizeCrowdsaleCommand = async (command, state) => {
     let finalizeTx = await state.crowdsaleContract.finalize({from: account});
 
     let fundsRaised = state.weiRaised.div(state.weiPerUSDinTGE),
-      minimumForMarketMaker = await state.crowdsaleContract.maxFoundationCapUSD;
+      minimumForMarketMaker = await state.crowdsaleContract.maxFoundationCapUSD.call();
```

Comment:
This should be calculated with the `state.weiPerUSDinTGE*10000000` ?

Reply:
No, because it's about the `maxFoundationCapUSD` constant defined in the contract. This value is not defined in the test state AFAIK

Reply:
https://github.com/windingtree/LifToken/blob/master/test/commands.js#L246 it is, and this value can change depending the weiPerUSD, we have the value in the state, we can use it.

Reply:
My bad I thought we were comparing the funds in wei, but its on usd. Its ok.

Reply:
no problem, thanks!